public/portfolio_shared_data.json

```diff
@@ -1,7 +1,7 @@
 {
     "basic_info": {
-      "name": "Davina Griss",
-      "titles": [ "Front-end Developer", "Senior Data Engineer", "Dev Team lead", "Mobile App Developer" ],
+      "name": "David Matalik",
+      "titles": [ "Web Developer", "React Creator", "TDD Enthusiast", "SOLID Lover"],
```

Comment:
what do you think about adding "Git Conquerer", "Scrum Explorer", and "XYZ Coach"?

Reply:
Yeah great idea, I will add "Git Conquerer" and "Scrum Explorer".

Reply:
:+1: